### DISCUSSION TITLE
I deleted the dreambooth models, but they still appear in the LORA model options, how to delete them?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [d8ahazard/sd_dreambooth_extension#551] [Q&A] [answered]
I deleted the dreambooth models, but they still appear in the LORA model options, how to delete them?

**Comment by JashoBell** [comment, marked as answer]
The Lora models (.pt files) are in a different place, check ./models/lora

**Reply by a-l-e-x-d-s-9** [reply]
Thank you very much!
